app/src/main/java/org/sopt/pingle/presentation/ui/main/more/MoreFragment.kt

```diff
@@ -87,7 +87,7 @@ class MoreFragment : BindingFragment<FragmentMoreBinding>(R.layout.fragment_more
             when (withDrawState) {
                 is UiState.Success -> {
                     kakaoAuthService.withdrawKakao()
-                    moveToSign()
+                    moveToOnboardingExplanation()
```

Comment:
훨씬 직관적이네요
```suggestion
                   navigateToOnboardingExplanation()
```
다른부분들에서 네이밍을 move 대신 navigate로 하니깐 요렇게 하는게 어떨까요?

Reply:
넹 다 navigate로 통일했습니다 ~!